### PR TITLE
AB#39084 GET /teams/{id}/channels/{id}/messages

### PR DESCRIPTION
**Pull request by lhao03**
Tested by using `?devx-api=https://graphexplorerapi.azurewebsites.net&org=LokiLAbs&branchName=interns/lhao03/39084-/teams/{id}/channels/{id}/messages`

### CODE REVIEW

**Comment by lhao03** [comment]
tested by adding to SampleQueries list, as right now can't switch to TeamsAppSampleQueries list atm.

**Review by omersayshi — APPROVED** [review]
The JSON is pushed to the right, besides everything looks good

File: sample-queries/sample-queries.json [hunk 1]
```diff
@@ -2983,6 +2983,15 @@
...
   "TeamsAppSampleQueries": [
-      
+            {
+                "id": "fe15123a-507a-45d8-aa29-e5b25f5c0ac7",
+                "category": "Microsoft Teams",
+                "method": "GET",
+                "humanName": "list channel messages",
+                "requestUrl": "/beta/teams/{team-id}/channels/{channel-id}/messages",
+                "docLink": "https://docs.microsoft.com/en-us/graph/api/channel-list-messages?view=graph-rest-beta&tabs=http",
+                "tip": "This query requires a team id and a channel id. To find the team id of teams you belong to, you can switch to user (delegated permissions) mode and run: GET https://graph.microsoft.com/v1.0/me/joinedTeams. To find the channel id of channels you belong to, stay in user (delegated permissions) mode and run: GET https://graph.microsoft.com/beta/teams/{team-id}/channels. Also ensure that the ChannelMessage.Read.Group permission is in your manifest.json.",
```
Comment (omersayshi) [inline sample-queries/sample-queries.json]: This sentence isnt really applicable 'Also ensure that the ChannelMessage.Read.Group permission is in your manifest.json.'

**Review by gracet9182 — APPROVED** [review]
╰(*°▽°*)╯

**Pull request merged**